projects/to_contribute/contribute_ideas.md

```diff
@@ -0,0 +1,51 @@
+## Project Contributions
```

Comment:
Same for this file, just pick one project to contribute to for now, e.g.

`projects/python/superstore.md`